swaap/swaap/Views/ProfileCardView.swift

```diff
@@ -161,12 +165,54 @@ class ProfileCardView: IBPreviewView {
 		jobTitle = userProfile?.jobTitle
 		location = userProfile?.location
 		industry = userProfile?.industry
+		tagline = userProfile?.tagline
+
+		lackOfInfoDescLabel.isHidden = true
+		hideUnhideElements()
 
 		guard let pContact = userProfile?.profileContactMethods.preferredContact else { return }
 		let nuggetInfo = ProfileInfoNugget(type: pContact.type, value: pContact.value)
 		preferredContact = nuggetInfo
 	}
 
+	private func hideUnhideElements() {
+		if jobTitle != nil {
+			jobTitleLabel.isHidden = false
+		} else {
+			jobTitleLabel.isHidden = true
+		}
+
+		if industry != nil {
+			industryStackView.isHidden = false
+		} else {
+			industryStackView.isHidden = true
+		}
+
+		if tagline != nil {
+			taglineContainer.isHidden = false
+		} else {
+			taglineContainer.isHidden = true
+		}
+
+		if location != nil {
+			if UIScreen.main.bounds.height > 667 {
+				locationStackView.isHidden = false
+			}
+		} else {
+			locationStackView.isHidden = true
```

Comment:
`.isVisible` again

Reply:
or better yet might be to avoid the if statements and instead just directly set `isVisible` or `isHidden` (whichever is better) directly from the nil/isEmpty value

Reply:
(there's some inconsistent behavior as you edit and remove different fields - create a tagline and then delete it, for example)